deploy/openshift/sws.yaml

```diff
@@ -9,6 +9,9 @@ parameters:
 - description: The version of the image to use
   name: IMAGE_VERSION
   value: dev
+- description: The namespace to use
+  name: NAMESPACE
+  value: default
```

Comment:
default should be istio-system, correct?

Reply:
+1 yeah I changed in the makefile but it needs to be changed here.